gradio/components/file.py

```diff
@@ -139,15 +141,36 @@ def preprocess(
             return [self._process_single_file(f) for f in payload]  # type: ignore
         return [self._process_single_file(payload)]  # type: ignore
 
+    def _download_files(self, value: str | list[str]) -> str | list[str]:
```

Comment:
Was debating whether this actually needed to be in the component since `gr.Blocks` can download files for you in the postprocess. But we need this here for the size of the file to be in the payload. LGTM.

Reply:
Yes exactly!